concordium-sdk/src/main/java/com/concordium/sdk/responses/modulelist/ModuleRef.java

```diff
@@ -0,0 +1,21 @@
+package com.concordium.sdk.responses.modulelist;
+
+import com.concordium.sdk.serializing.JsonMapper;
+import com.concordium.sdk.transactions.Hash;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import concordium.ConcordiumP2PRpc;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ModuleRef {
```

Comment:
I think you want to wrap a `Hash` ?
I.e. a module reference is simply a sha256 hash.

Reply:
done